blocks/subscriptions-block/features/paywall/default.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { useIdentity } from '@wpmedia/identity-block';
+import './styles.scss';
+
+const Paywall = ({ customFields }) => {
+  const { signUpURL, loginURL } = customFields;
+  const { Identity } = useIdentity();
+  const [showPaywall, setShowPaywall] = useState(false);
+  const [loggedIn, setIsLoggedIn] = useState(false);
+
+  useEffect(() => {
```

Comment:
The `useEffect`s in here are for demo purpose. Paywall is another concept item we is too be looked at